scripts/jenkins/ed.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
And if it's more than just `cat $1 > $2` we probably want to name the variables to express their content.